src/index.ts

```diff
@@ -60,6 +61,22 @@ export class Spinner {
     this.refresh();
   }
 
+  private onProcessExit = () => {
```

Comment:
After overriding the termination signal handlers, node won't exit on it's own so we need to call process.exit with the correct exit code after handling the signals.

Reply:
Good catch! I'll have a look